scripts/python/step_2.py

```diff
@@ -1,105 +1,138 @@
+  'Circle-Token': getenv("MIGRATION_CLOUD_TOKEN")
+}
+
+onPremHeaders = {
+  'Content-Type': 'application/json',
+  'Accept': 'application/json',
+  'Circle-Token': getenv("MIGRATION_SERVER_TOKEN")
+}
 
-def cloneRepo(gitClonePath):
-    """Clone the repo and checkout the default branch"""
+def deleteBranch(branch):
+    """Delete the temporary branch we created in step_1.py, get-secrets"""
     try:
-        repo = Repo.clone_from(
-            'https://{}:@github.com/{}/{}.git'.format(getenv("MIGRATION_GITHUB_TOKEN"),\
-                 org, project), gitClonePath,
-            branch='master'
-        )
-        print("Cloned the repo {}/{} into {}".format(org, project, gitClonePath))
-        return repo
+        g = Github(getenv("MIGRATION_GITHUB_TOKEN"))
+        repo = g.get_repo('{}/{}'.format(org, project))
+        if repo:
+            print('Attempting to delete branch {} in project {}/{}'.format(branch, org, project))
+            repo.get_git_ref('heads/{}'.format(branch)).delete()
+            print('Deleted branch {} in project {}/{}'.format(branch, org, project))
     except Exception as e:
-        print(e)
         return SystemExit(e)
 
-def updateClone(gitClonePath):
+def getCSV(project):
     """
-    Copy scripts/python into the git clone dir
-    And copy the circle config file into the git clone dir
-    Returns a list of dirs to run `git add` on.
+    Download the JSON file containing the project's secret key:val pairs
+    from the S3 bucket.
     """
     try:
-        currDir = getcwd()
-        # Copy scripts/python/ci_migration_script.py into the git clone dir under scripts/python/ci_migration_script.py
-        scriptPath = path.join(currDir, "ci_migration_script.py")
-        scriptDir = reduce(path.join,[gitClonePath, 'scripts', 'python'])
-        makedirs(scriptDir)
-        copy(scriptPath, scriptDir)
-        print("Copied scripts/python/ci_migration_script.py into {}/scripts/python/ci_migration_script.py".format(gitClonePath))
-    
-        # Copy example-config/.circleci/config.yml into the git clone dir under .circleci/config.yml
-        circlePath = reduce(path.join,[str(Path(getcwd()).parents[1]), 'example-config', '.circleci', 'config.yml'])
-        circleDir = reduce(path.join,[gitClonePath, '.circleci'])
-        createDir(circleDir)
-        copy(circlePath, circleDir)
-        print("Copied example-config/.circleci/config.yml into {}/.circleci/config.yml".format(gitClonePath))
-        return [circleDir, scriptDir]
-    except Exception as e:
-        return SystemExit(e)
+        s3client = client('s3', aws_access_key_id=getenv("MIGRATION_AWS_ACCESS_KEY_ID"),aws_secret_access_key=getenv("MIGRATION_AWS_SECRET_ACCESS_KEY"))
+        bucket = getenv("MIGRATION_BUCKET")
+        filepath = '{}/{}.json'.format(getenv("MIGRATION_PREFIX"), project)
+        filename = "{}.json".format(project)
+        s3client.download_file(bucket, filepath, '{}.json'.format(project))
```

Comment:
I noticed this kept files on disk at `scripts/python/<project>.json`. Might be good to clean this up after it's uploaded.

Reply:
Thank you!